src/actions/getActiveSubscription.ts

```diff
@@ -10,10 +10,14 @@ type PricePlan = components['schemas']['PricePlan'];
 export default function getActiveSubscription(
   customerToken: string
 ): Promise<PricePlan | null> {
+  if (!customerToken) {
+    throw new Error('Token must be provided.');
+  }
```

Comment:
Nice (I'd hope TypeScript would catch this, but I guess not everyone uses TypeScript).